assets/js/blocks/customer-account/style.scss

```diff
@@ -34,6 +34,7 @@
 		}
 
 		.wc-block-customer-account__account-icon {
+			box-sizing: content-box !important;
```

Comment:
Is the `!important` necessary?

It seems to be working correctly for me  without it:

<img width="1184" alt="Customer_Account_–_productgallery" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/6acf6be9-7df5-4e1f-9e24-352a8c1d7478">

Reply:
Yea, it's kinda necessary because it depends on the theme. In this case, it works because the theme doesn't define that property. If the theme were to define that (and keep in mind that it is an inheritable property, so it could cascade down easily from something unrelated) that would override it.

Reply:
You're right in being cautious! And I'm the same. I hate `!important`. However, in certain cases it is the only right tool. I try to use it as specifically as possible, like this case.

Reply:
Hey @danieldudzic if that makes sense to you, would you feel like approving this? Or are there any other comments to address?

Reply:
Thank you for addressing my concern 🙏